style/productpage.css

```diff
@@ -0,0 +1,168 @@
+
+:root{
+    --primary-color:black;
+    --secondary-color:tan;
+    --active-color:aqua;
+}
+body{
+    background-color: tan;
+    padding: 0%;
+    border: 0%;
```

Comment:
instead of applying rest CSS on body, you can add it to a universal selector

Reply:
ok in root